src/main/scala/strawman/collection/Set.scala

```diff
@@ -76,38 +76,7 @@ trait SetMonoTransforms[A, +Repr]
 
 }
 
-trait SetPolyTransforms[A, +C[X]] extends IterablePolyTransforms[A, C] {
-
-  /** Creates a new $coll by adding all elements contained in another collection to this $coll, omitting duplicates.
-    *
-    * This method takes a collection of elements and adds all elements, omitting duplicates, into $coll.
-    *
-    * Example:
-    *  {{{
-    *    scala> val a = Set(1, 2) concat Set(2, 3)
-    *    a: scala.collection.immutable.Set[Int] = Set(1, 2, 3)
-    *  }}}
-    *
-    *  @param that     the collection containing the elements to add.
-    *  @return a new $coll with the given elements added, omitting duplicates.
-    */
-  def concat(that: Set[A]): C[A]
-
-  /** Alias for `concat` */
-  @inline final def ++ (that: Set[A]): C[A] = concat(that)
-
-  /** Computes the union between of set and another set.
-    *
-    *  @param   that  the set to form the union with.
-    *  @return  a new set consisting of all elements that are in this
-    *  set or in the given set `that`.
-    */
-  @inline final def union(that: Set[A]): C[A] = concat(that)
-
-  /** Alias for `union` */
-  @inline final def | (that: Set[A]): C[A] = concat(that)
-
-}
+trait SetPolyTransforms[A, +C[X]] extends IterablePolyTransforms[A, C]
```

Comment:
It is "essentially impossible to do wrong" (by accident) as modified, which is a good thing.  But if you actually want the concat--perhaps because you're abstracting over both mutable and immutable--it's a much bigger hassle, which is bad.  I'm not convinced the good outweighs the bad.